src/Forms/Android/ArcGISRuntime.Xamarin.Forms.Android.csproj

```diff
@@ -23,7 +23,7 @@
     <MandroidI18n />
     <JavaMaximumHeapSize />
     <JavaOptions />
-    <TargetFrameworkVersion>v8.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v10.0</TargetFrameworkVersion>
```

Comment:
Does this change need to be made for native Android as well?

Reply:
This can't be done with the current android support libraries, I will add the android x libraries to this branch.